<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task_managerClass</class>
 <widget class="QMainWindow" name="Task_managerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task_manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QListWidget" name="repo_list"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="total_duration_list"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_priority">
          <property name="text">
           <string>Priority:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="priority_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="duration_button">
        <property name="text">
         <string>Show duration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Task_manager.qrc"/>
 </resources>
 <connections/>
</ui>
